entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -181,27 +190,34 @@ class EntityService(
         values.filterValues {
             it[0] is Map<*, *>
         }.mapValues {
-            expandValueAsMap(it.value)
+            expandValueAsListOfMap(it.value)
         }.filter {
             !(NGSILD_PROPERTIES_CORE_MEMBERS.plus(NGSILD_RELATIONSHIPS_CORE_MEMBERS)).contains(it.key) &&
                 isAttributeOfType(it.value, NGSILD_PROPERTY_TYPE)
         }.forEach { entry ->
-            logger.debug("Creating property ${entry.key} with values ${entry.value}")
-
-            // for short-handed properties, the value is directly accessible from the map under the @value key
-            val propertyValue =
-                getPropertyValueFromMap(entry.value, NGSILD_PROPERTY_VALUE) ?: entry.value["@value"]!!
-
-            val rawProperty = Property(
-                name = entry.key, value = propertyValue,
-                unitCode = getPropertyValueFromMapAsString(entry.value, NGSILD_UNIT_CODE_PROPERTY),
-                observedAt = getPropertyValueFromMapAsDateTime(entry.value, NGSILD_OBSERVED_AT_PROPERTY)
-            )
+            entry.value.forEach { instance ->

```

Comment:
As I understand the spec, multi-attribute applies to entities, not to attributes.

Reply:
Are you referring to that ? 
"For each Entity, there can be Attributes that simultaneously have more than one instance."

Reply:
Indeed